kv/txn_test.go

```diff
@@ -251,12 +251,12 @@ func TestTxnDBUncertainty(t *testing.T) {
 	// Make sure that we notice immediately if any kind of backing off is
 	// happening. Restore the previous options after this test is done to avoid
 	// interfering with other tests.
-	defaultRetryOptions := client.TxnRetryOptions
+	defaultRetryOptions := client.DefaultTxnRetryOptions
```

Comment:
I'd love to eliminate this messy pattern if we can just get ahold of the client.KV.Context.TxnRetryOptions in verifyUncertainty().